test/clients/test_clients.py

```diff
@@ -14,6 +14,10 @@ def test_it_contains_clients():
 
     assert len(clients) == 2
 
+    for client in clients:
+        assert isinstance(client, Client)
+        assert client in clients
```

Comment:
Probably want to avoid the `for` (bit redundant, and looks odd), and add a negative assertion too.